### DISCUSSION TITLE
An idea on reworking captain's talent "Down with the ship"

### DISCUSSION THREAD

**Discussion by BiffW** [FakeFishGames/Barotrauma#10285] [Ideas and suggestions]
**Why I don't like it now**
Currently it boosts captain's ability to fix and repair which are obviously mechanic and engineer's job.
It's weird to encourage a captain to do fixing and repairing.
And it feels negative because it only gives bonus when the sub is flooded more than 50%.

**It can be a quality change if reworked like this**
Down with the Ship
As long as you are in your submarine, you will gain immunity to stun and remain fully conscious even if your vitality is below 0.

**Why**
This allows them to give commands and useful informations to their teammates when attacked by stunning projectiles from giant spinelings, latchers or some powerful mod monsters, which can be an advantage in multiplayer.
I believe it is captain's job to navigate the submarine and give commands, infromations to other teammates.
And practically this shouldn't be difficult to realize since you can copy codes from Security's tier 4 talent on middle tree.

**Balance**
I believe it's not too OP since it only works when captain is in the sub.

**Comment by MarkCringan** [comment]
So, the captain is essentially immortal whilst in the submarine? Or are you suggesting they just be immune to stun?

**Reply by BiffW** [reply]
Sorry for being confusing.
I'm updating it in barotrauma language.

**Reply by MapleWheels** [reply]
> Sorry for being confusing. I'm updating it in barotrauma language.

Even with the changes, still makes the captain too strong in the sub.

**Comment by MapleWheels** [comment]
Down With the Ship is basically mandatory on ships with small crews. When you go under 4-5 players, the Captain ends up doing double duty as Security and Mechanic during high-workload situations.

I've been seeing a lot of talk about talents in large crews but no one is giving any mind it seems to small crew playthroughs, which btw, is what makes up most friends' groups playthroughs.

**Reply by NonCancerAlias** [reply]
Maple wheels hit the nail on the head, the talent system is supposed to offer one the tools for most circumstances and you pick accordingly to what your circumstances are or what you plan for them to be.
